### PR TITLE
[INTEROP-7356] PoC to enable FIPS testing on OCP 4.16

### PR DESCRIPTION
**Pull request by dfrazzette**
PoC to configure OCP 4.16 tests to run w/FIPS enabled (Dev Spaces, ODF, Quay).

### CODE REVIEW

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-redhat-developer-devspaces-interop-tests-main-devspaces-ocp4.16-lp-interop-devspaces-interop-aws-fips

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-redhat-developer-devspaces-interop-tests-main-devspaces-ocp4.16-lp-interop-devspaces-interop-aws-fips

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-redhat-developer-devspaces-interop-tests-main-devspaces-ocp4.16-lp-interop-devspaces-interop-aws-fips

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-redhat-developer-devspaces-interop-tests-main-devspaces-ocp4.16-lp-interop-devspaces-interop-aws

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-red-hat-storage-ocs-ci-master-odf-ocp4.16-lp-interop-odf-interop-aws-fips periodic-ci-quay-quay-tests-master-ocp-416-quay-quay-e2e-tests-quay311-ocp416-lp-interop-fips

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-red-hat-storage-ocs-ci-master-odf-ocp4.16-lp-interop-odf-interop-aws periodic-ci-quay-quay-tests-master-ocp-416-quay-quay-e2e-tests-quay311-ocp416-lp-interop

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-red-hat-storage-ocs-ci-master-odf-ocp4.16-lp-interop-odf-interop-aws-fips periodic-ci-quay-quay-tests-master-ocp-416-quay-quay-e2e-tests-quay311-ocp416-lp-interop-fips periodic-ci-redhat-developer-devspaces-interop-tests-main-devspaces-ocp4.16-lp-interop-devspaces-interop-aws-fips

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-redhat-developer-devspaces-interop-tests-main-devspaces-ocp4.16-lp-interop-devspaces-interop-aws-fips

**Comment by liswang89** [comment]
For bot jobs, ocp-self-managed-lp-watcher-bot-message-416-fips, you need the similar cron config but maybe a few hours later to allow tests to complete.

**Comment by dfrazzette** [comment]
> For bot jobs, ocp-self-managed-lp-watcher-bot-message-416-fips, you need the similar cron config but maybe a few hours later to allow tests to complete.

@liswang89 - There will be only 1 self-managed test run triggered on any given Monday (_either_  FIPS-enabled _or_ non-FIPS [standard], not both):
1st Monday: FIPS-enabled
2nd Monday: non-FIPS
3rd Monday: non-FIPS
4th Monday: non-FIPS
_5th Monday: non-FIPS_ 

Because there will be no concurrent self-managed runs, the watcher bot job can trigger the same time each Monday.

File: ci-operator/config/red-hat-storage/ocs-ci/red-hat-storage-ocs-ci-master__odf-ocp4.16-lp-interop.yaml [hunk 1]
```diff
@@ -67,6 +67,50 @@ tests:
...
+            ],
+          "success_rules":
+            [
+              {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
+            ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.16-lp","self-managed-lp","fips","odf-lp"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      ODF_VERSION_MAJOR_MINOR: "4.15"
```
Comment (Simranpal) [inline ci-operator/config/red-hat-storage/ocs-ci/red-hat-storage-ocs-ci-master__odf-ocp4.16-lp-interop.yaml:100]: Could you verify if this is the correct/latest version of odf to use here and its support period

Comment (dfrazzette) [reply ci-operator/config/red-hat-storage/ocs-ci/red-hat-storage-ocs-ci-master__odf-ocp4.16-lp-interop.yaml:100]: @Simranpal - According the Portfolio Product Pages, [ODF 4.15](https://pp.engineering.redhat.com/odf-4.15/overview/) is latest release (ODF 4.16 release targeted for mid-July). This version is what has been currently tested in OCP 4.16 self-managed.

**Review by Simranpal — APPROVED** [review]
/lgtm

**Pull request merged**